src/ui/components/Popup/Popup.test.jsx

```diff
@@ -0,0 +1,19 @@
+import { render, fireEvent, screen } from '@testing-library/react';
+import { Popup } from './Popup';
+
```

Comment:
Możesz jeszcze dodać test, gdy `open={false}` to `Popup` nie wyświetla nic.

Reply:
Dodałam, mam nadzieję że o to chodziło :D